### PR TITLE
[WIP] New Travis build

### PR DESCRIPTION
**Pull request by prisis**

### CODE REVIEW

**Comment by Nyholm** [comment]
We should not allow HHVM to fail.

**Comment by prisis** [comment]
I will change it

**Comment by Nyholm** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -14,11 +14,11 @@ php:
 
 matrix:
   include:
-    - php: 5.6
+    - php: 7.0
       env: SEND_COVERAGE=true
+    - php: 7.0
+      env: SUBSPLIT=true
```
Comment (Nyholm) [inline .travis.yml]: Do this on the same build as SEND_COVERAGE

**Comment by Nyholm** [comment]
Do not forget to add 
```yaml
after_success:
    - codecov # If coverage
```

**Pull request closed without merging**